server/src/utils.js

```diff
@@ -0,0 +1,64 @@
+const {DataSource} = require('apollo-datasource');
+
+const createStore = () => {
+  // Fake DB
+  // mockup data
+  // todo: move this into separated CSV file
+  //  and import it into DB
+
+  const users = [
```

Comment:
This type seems to express 1 single user. We should call it as "user" then.

Reply:
But it is an array. Someday more users might be added. Isn't it more appropriate to leave it as 'users', I think?

Reply:
Then it's a little confusing why users[], events[], and tags[] are defined at different levels (i.e. 1 event can have multiple participants and multiple tags). 

Instead of clarifying the data schema here, let's do the schema design separately in the db, and this script should just reflect what's stored on db. As you noted in the TODO, we can just remove this script from this PR, and re-merge when DB is ready.